ert_shared/ensemble_evaluator/ensemble/prefect.py

```diff
@@ -220,16 +220,16 @@ def context_method():
     def evaluate(self, config: EvaluatorServerConfig, ee_id: str):
         self._ee_id = ee_id
         self._ee_config = config
-        # self._eval_proc = threading.Thread(
-        #     target=self._evaluate,
-        #     args=(config, ee_id),
-        # )
-
-        mp_ctx = multiprocessing.get_context(method=PrefectEnsemble.context_method())
-        self._eval_proc = mp_ctx.Process(
+        self._eval_proc = threading.Thread(
             target=self._evaluate,
             args=(config, ee_id),
         )
+
+        # mp_ctx = multiprocessing.get_context(method=PrefectEnsemble.context_method())
+        # self._eval_proc = mp_ctx.Process(
+        #     target=self._evaluate,
+        #     args=(config, ee_id),
+        # )
         self._eval_proc.daemon = True
```

Comment:
Is there a reason why are we changing this? 🤔

Reply:
Currently, the change is just tentative we might need to make it permanent if we want to do any kind of posting data from the prefect workers to storage using the async client `httpx`.

But currently, it requires a bit more testing and investigation